run_benchmarks.sh

```diff
@@ -0,0 +1,28 @@
+TMPDIR="${PWD}/bundles"
```

Comment:
@milkysoft it'd be a good idea to exit the script on error by adding `set -e` at the beginning: https://stackoverflow.com/a/19622569

Reply:
@g4s8 , fixed with latest [commit](https://github.com/artipie/rpm-adapter/pull/386/commits/7607ca7c554416dcdf9bcb7723af6fc9e1f3c09b)